File: C/Assignment-2/Prog2.c
#include<stdio.h>
#include<math.h>
int main(){
    int x1, y1, x2, y2;
    printf("Enter x1 and y1 : ");
    scanf("%d %d", &x1, &y1);
    printf("Enter x2 and y2 : ");
    scanf("%d %d", &x2, &y2);
    
    float distance = sqrt(pow((x1-x2), 2) + pow((y1-y2), 2));
    printf("Distance : %f", distance);
    return 0;
}

File: Python/Assignment-2/Prog2.py
import math

x1, y1 = map(float, input("Enter the co-ordinates for the 1st point : ").split())
x2, y2 = map(float, input("Enter the co-ordinates for the 2nd point : ").split())

result = math.sqrt((x1-x2)**2 + (y1-y2)**2)
